examples/warntoggle/mqttwarn/customfunctions.py

```diff
@@ -14,7 +14,7 @@ def togglestate(topic, payload, section, srv):
         if topic in toggles:
             # file found, topic found
             topicblock = toggles[topic]
-	        srv.logging.debug('togglestate() was called from the ' + section + ' section and found ' + topic + ' in ' + filename)
+	    srv.logging.debug('togglestate() was called from the ' + section + ' section and found ' + topic + ' in ' + filename)
```

Comment:
it looks to me as though there are a few spaces missing there ... the `srv.` ought to be under `topicblock`